docs/content/commands/update.md

```diff
@@ -65,3 +65,10 @@ If you want to update a single group you can use the following command:
   `--redirects`: Creates binding redirects for the NuGet packages.
 
   `--no-install`: Skips paket install --hard process afterward generation of [`paket.lock` file](lock-file.html).
+
+## Updating http dependencies
+
+The update command has no effect on http dependencies. If you want to update a file you need to use the [`paket install` command](paket-install.html) with `--force [-f]` option.
```

Comment:
I do not see it as a bug. It is not straightforward to determine if a file is outdated through http unless you allways download it and store a hash of it. You would need a system based on http headers and I doubt it could be universally reliable. 
It would be a bit overkill to download every file to test for updates. I find it a good trade-off to be aware of that and need to force the install.